x/group/simulation/genesis.go

```diff
@@ -0,0 +1,189 @@
+			GroupVersion:        uint64(i + 1),
+			GroupAccountVersion: uint64(i + 1),
+			Status:              group.ProposalStatusSubmitted,
+			Result:              group.ProposalResultAccepted,
+			VoteState: group.Tally{
+				YesCount:     "3",
+				NoCount:      "0",
+				AbstainCount: "0",
+				VetoCount:    "0",
+			},
+			ExecutorResult: group.ProposalExecutorResultNotRun,
+			Metadata:       []byte(simtypes.RandStringOfLength(r, 50)),
+			SubmittedAt:    gogotypes.Timestamp{Seconds: 1},
+			Timeout:        gogotypes.Timestamp{Seconds: 1000},
+		}
+		err := proposal.SetMsgs([]sdk.Msg{&banktypes.MsgSend{
+			FromAddress: from.Address.String(),
+			ToAddress:   to.Address.String(),
+			Amount:      sdk.NewCoins(sdk.NewInt64Coin("test", 10)),
+		}})
+		if err != nil {
+			panic(err)
+		}
+
+		proposals[i] = proposal
+	}
+	return proposals
+}
+
+func getVotes(r *rand.Rand, simState *module.SimulationState) []*group.Vote {
+	votes := make([]*group.Vote, 3)
+
+	for i := 0; i < 3; i++ {
+		votes[i] = &group.Vote{
+			ProposalId:  uint64(i + 1),
+			Voter:       simState.Accounts[i].Address.String(),
+			Choice:      group.Choice_CHOICE_YES,
+			Metadata:    []byte(simtypes.RandStringOfLength(r, 50)),
+			SubmittedAt: gogotypes.Timestamp{Seconds: 10},
+		}
+	}
+
+	return votes
+}
+
+// RandomizedGenState generates a random GenesisState for the group module.
+func RandomizedGenState(simState *module.SimulationState) {
+
+	// groups
+	var groups []*group.GroupInfo
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, GroupInfo, &groups, simState.Rand,
+		func(r *rand.Rand) { groups = getGroups(r, simState.Accounts) },
+	)
+
+	// group members
+	var members []*group.GroupMember
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, GroupMembers, &members, simState.Rand,
+		func(r *rand.Rand) { members = getGroupMembers(r, simState.Accounts) },
+	)
+
+	// group accounts
+	var groupAccounts []*group.GroupAccountInfo
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, GroupAccountInfo, &groupAccounts, simState.Rand,
+		func(r *rand.Rand) { groupAccounts = getGroupAccounts(r, simState) },
+	)
+
+	// proposals
+	var proposals []*group.Proposal
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, GroupProposals, &proposals, simState.Rand,
+		func(r *rand.Rand) { proposals = getProposals(r, simState) },
+	)
+
+	// votes
+	var votes []*group.Vote
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, GroupVote, &votes, simState.Rand,
+		func(r *rand.Rand) { votes = getVotes(r, simState) },
+	)
+
+	groupGenesis := group.GenesisState{
+		GroupSeq:        1,
+		Groups:          groups,
+		GroupMembers:    members,
+		GroupAccountSeq: 1,
+		GroupAccounts:   groupAccounts,
+		ProposalSeq:     1,
+		Proposals:       proposals,
+		Votes:           votes,
+	}
+
+	bz, err := simState.Cdc.MarshalJSON(&groupGenesis)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Printf("Selected randomly generated %s parameters:\n%s\n", group.ModuleName, bz)
```

Comment:
```suggestion
	fmt.Printf("selected randomly generated %s parameters:\n%s\n", group.ModuleName, bz)
```

Reply:
This is genesis state and not just parameters right?
```suggestion
	fmt.Printf("selected randomly generated %s genesis state:\n%s\n", group.ModuleName, bz)
```